multiapps-controller-core/src/test/java/org/cloudfoundry/multiapps/controller/core/helpers/common/AbstractConfigurationReferencesResolverTest.java

```diff
@@ -0,0 +1,138 @@
+package org.cloudfoundry.multiapps.controller.core.helpers.common;
+
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.Map;
+
+import org.cloudfoundry.multiapps.common.test.TestUtil;
+import org.cloudfoundry.multiapps.common.test.Tester;
+import org.cloudfoundry.multiapps.common.test.Tester.Expectation;
+import org.cloudfoundry.multiapps.common.util.JsonUtil;
+import org.cloudfoundry.multiapps.common.util.YamlParser;
+import org.cloudfoundry.multiapps.controller.core.helpers.v2.ConfigurationFilterParser;
+import org.cloudfoundry.multiapps.controller.core.helpers.v2.ConfigurationReferencesResolver;
+import org.cloudfoundry.multiapps.controller.core.test.MockBuilder;
+import org.cloudfoundry.multiapps.controller.core.util.ApplicationConfiguration;
+import org.cloudfoundry.multiapps.controller.persistence.model.CloudTarget;
+import org.cloudfoundry.multiapps.controller.persistence.model.ConfigurationEntry;
+import org.cloudfoundry.multiapps.controller.persistence.model.filters.ConfigurationFilter;
+import org.cloudfoundry.multiapps.controller.persistence.query.ConfigurationEntryQuery;
+import org.cloudfoundry.multiapps.controller.persistence.services.ConfigurationEntryService;
+import org.cloudfoundry.multiapps.mta.builders.v2.ParametersChainBuilder;
+import org.cloudfoundry.multiapps.mta.handlers.ConfigurationParser;
+import org.cloudfoundry.multiapps.mta.handlers.v2.DescriptorParser;
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Platform;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.mockito.Answers;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+public abstract class AbstractConfigurationReferencesResolverTest {
```

Comment:
Why this class is marked as abstract without any abstract methods? Can we just call it base class for other test classes?